### ISSUE TITLE
results of pre-trained model

### ISSUE THREAD

**Issue by XJTUbopenghui** [JiaRenChang/PSMNet#122]
I ran your pre-trained models on Scene Flow and KITTI dataset. And I found that the error of Pre-trained SceneFlow model is much larger than the result in your paper.
And I also found that the training loss is relatively large for your pre-trained model on KITTI dataset, while the 3-pix error is very small.
I trained from scratch by myself on KITTI dataset. But the training loss for my model is very small, the 3-pix error is much larger than yours.
Is there any mistakes I made in the training?
thank you!

**Comment by shidongjian** [comment]
hello what does 3-px error mean？

**Comment by Longseabear** [comment]
3-pixel error = pixel ratio of [ abs(predict_disparity-ground_disparity) > 3 ]
